src/main/java/net/spy/memcached/MemcachedConnection.java

```diff
@@ -330,12 +333,14 @@ private void handleNodesToRemove(final List<MemcachedNode> nodesToRemove) {
       }
       /* ENABLE_MIGRATION end */
 
+      if (node.isActive()) {
```

Comment:
제생각에는 `if (!node.isActive())`일 때에만 cancelAllOperations 호출하는 형태가 더 이해하기 쉽고 필요한 주석도 줄어들 것으로 보이는데 어떤가요?

Reply:
https://github.com/naver/arcus-java-client/pull/749#discussion_r1584005999

주석은 필요하리라 여겨집니다.

Reply:
주석의 양이 줄어든다는 의미였습니다.

Reply:
추가된 주석은 한 줄인데, 어떻게 더 줄일 수 있나요?

Reply:
아래와 같이 작성하면 더 깔끔해질 것 같다는 의미이며, 생각해보니 체감할만큼 줄어들지는 않네요.
딱히 중요하진 않아서 넘기셔도 됩니다.

```java
// if memcached node is removed from ZK and closed connection, 
// cancel operations regardless of failure mode because removing node is not related to failure mode.
if (!node.isActive()) {
  cancelAllOperations(node, "node removed.");
}
```